plugins.xml

```diff
@@ -7173,6 +7173,52 @@
 			<institution>Reviewer Credits GmbH</institution>
 			<email>emre.danisan@reviewercredits.com</email>
 		</maintainer>
+		
+		<release date="2025-03-20" version="3.3.0.9" md5="4618a32978c65733f8b6e65b5de78572">
+			<package>https://www.reviewercredits.com/wp-content/uploads/reviewer-credits/ojsplugin/V3009/reviewerCredits.tar.gz</package>
+			<compatibility application="ojs2">
+				<version>3.3.0.0</version>
+				<version>3.3.0.1</version>
+				<version>3.3.0.2</version>
+				<version>3.3.0.3</version>
+				<version>3.3.0.4</version>
+				<version>3.3.0.5</version>
+				<version>3.3.0.6</version>
+				<version>3.3.0.7</version>
+				<version>3.3.0.8</version>
+				<version>3.3.0.9</version>
+				<version>3.3.0.10</version>
+				<version>3.3.0.11</version>
+				<version>3.3.0.12</version>
+				<version>3.3.0.13</version>
+				<version>3.3.0.14</version>
+				<version>3.3.0.15</version>
+				<version>3.3.0.16</version>
+				<version>3.3.0.17</version>
+				<version>3.3.0.18</version>
+				<version>3.3.0.19</version>
+				<version>~3.3.0.0</version>
+			</compatibility>
+			<certification type="partner"/>
+			<description>Plugin to enable the reviewer finder and peer review data integration with Reviewer Credits.</description>
+		</release>
+		<release date="2025-03-20" version="3.3.0.9" md5="6e59a10474d511297c62edfb37da1f03">
```

Comment:
I see this is also versioned `3.3.0.9` (but should be `3.0.0.9`). The two releases of the plugin -- for 3.3.0 and 3.4.0 -- should use different versions, and you should ensure that users upgrading will always be using a version of the plugin that's the same or higher than the last. (For example, someone using 3.0.0.9 with OJS 3.3.0-x who upgrades to OJS 3.4.0-x should find themselves using a higher version of your plugin.)

We tend to version plugins like this (the numbers are just examples):
- 1.2.x for OJS  3.3.0-x
- 1.3.x for OJS 3.4.0-x

That way you can release 1.2.1, 1.2.2, etc. of your plugin to fix bugs, without having to simultaneously release a package for OJS 3.4.0-x.